test/unit/ajax.js

```diff
@@ -1834,13 +1834,7 @@ QUnit.module( "ajax", {
 					},
 					"Etag": {
 						url: "mock.php?action=etag",
-
-						// Support: TestSwarm
-						// TestSwarm is now proxied via Cloudflare which cuts out
-						// headers relevant for ETag tests, failing them. We're still
-						// running those tests in Karma on Chrome & Firefox (including
-						// Firefox ESR).
-						qunitMethod: QUnit.isSwarm ? "skip" : "test"
+						qunitMethod: "test"
```

Comment:
You can just remove `qunitMethod` from here & above and use `QUnit.test` directly below. Also, with a single entry in the objects, you can now map to URLs directly, simplifying the logic below a bit.